topics/climate/tutorials/ocean_qcv_analysis/tutorial.md

```diff
@@ -42,308 +51,526 @@ This tutorial explains how to qualify and calibrate the nitrate sensor mounted o
+> - **Switch history**: click on the **⇄** icon and select a history
+> - **Copy datasets**: click on **Operations** → *Copy datasets* and choose source/destination histories
 >
-> The first time you use Galaxy, there will be no files in your history panel.
+> ![Galaxy operation symbol](../../images/bgc_calib/interface_2.png)
+{: .details}
+
+> <details-title>Galaxy execution constraints</details-title>
+> - **Quota per user account**: check [your quota](https://galaxyproject.org/support/account-quotas/).  
+> Free space by clicking the database icon in the history panel and “Review and clear” GB.  
+> (Note: deleting a dataset in the history does not free quota until you purge it.)
+> 
+> ![Galaxy quota info](../../images/bgc_calib/galaxy_storage_handling.png)
+> 
+> - **Time limit for interactive tools**: 24h. After that, ODV closes automatically.  
+> - **Avoid weekends**: upgrades are performed and can stop tools/workflows.  
+> - **Max interactive tools simultaneously**: 2.
+{: .details}
+
+## Nitrate calibration
+
+> <comment-title>Oxygen prerequisite</comment-title>
+> Because oxygen concentration is implied in the estimation of the *nitrate adjustment from neural network method*,  
+> platforms to be calibrated must have **good-quality oxygen** (adjusted in real time or delayed mode for Argo floats or gliders).
+{: .comment}
+
+> <details-title>World Ocean Atlas reference</details-title>
+> Nitrate calibration needs a potential reference dataset such as the **WOA annual climatology**.  
+> You can [download it here](https://www.ncei.noaa.gov/thredds-ocean/catalog/woa23/DATA/nitrate/netcdf/all/1.00/catalog.html?dataset=woa23/DATA/nitrate/netcdf/all/1.00/woa23_all_n00_01.nc) and upload it into Galaxy.
 {: .details}
 
-# Manage your data 
+# Manage your data
+
+## Step 1: Get your data 
+
+> <hands-on-title>Get your data from the S3 server</hands-on-title>
+> - Click on **Upload** in the vertical panel on the left
+> - In the pop-up window, click on **Choose from repository**
+> - In the top search bar, search for `argo`
+> - In the result list, select **Argo marine floats data and metadata from Global Data Assembly Centre (Argo GDAC)**
+> - Navigate to `pub/dac/coriolis/4903881`
+> - Tick the following:
+>   - `4903881_meta.nc`
+>   - All files under `profiles/`
+> - Click on **Select** (bottom-right)
+> - Remove the synthetic files `SR4903881_*.nc` (by clicking the trash icon)
+> - Click on **Start**
+> - Wait until all files are stored in your history on the right (green color)
+>
+> ![S3 upload interface](../../images/bgc_calib/s3_upload.png)
+{: .hands_on}
 
-> <hands-on-title>Get data</hands-on-title>
-> Upload the argo data files (meta, core and BGC) of the float 4903881 from the S3 service. 
-> - 2a. Click on “Upload” on the vertical panel on the left. A pop-up is launched
-> - 2b. Click on “Choose from repository” at the bottom of the popup
+> <hands-on-title>Get your data from your computer</hands-on-title>

```

Comment:
Are students supposed to do upload both from S3 and from their local computer for this tutorial? Or is it just extra info (in that case I would put it in a details box to make it clear it is just a tip for when they are doing their own analysis)